src/Helpers.cpp

```diff
@@ -42,3 +42,8 @@ void KerbalSimpit::setCameraMode(byte mode)
 {
   _send(CAMERA_CONTROL_MODE, &mode, 1);
 }
+
+void KerbalSimpit::cycleNavBallMode()
+{
+  _send(NAVBALLMODE_MESSAGE, 1, 1);
```

Comment:
This line send a payload of 1 byte and reads it from address 1. We don't know what is in this adress, so I believe the behavior is unexpected. I suggest to use instead this code, to always send 0 since the mod KSP side is not ready to handle empty messages : 

```
  byte payload = 0;
  _send(NAVBALLMODE_MESSAGE, &payload, 1);
```